.stylelintrc.json

```diff
@@ -4,6 +4,7 @@
     "color-named": null,
     "selector-max-id": null,
     "selector-no-qualifying-type": null,
-    "property-no-vendor-prefix": null
+    "property-no-vendor-prefix": null,
+    "selector-class-pattern": null
```

Comment:
also, i'm totally down to disable some linting commands, but I think I just ran linting, and I think it's just for 
```
 Selector should be written in lowercase with hyphens
```
which seems like an ok standard? Thoughts?